mypyc/test-data/run-dicts.test

```diff
@@ -91,6 +91,11 @@ od.move_to_end(1)
 assert get_content(od) == ([3, 1], [4, 2], [(3, 4), (1, 2)])
 assert get_content_set({1: 2}) == ({1}, {2}, {(1, 2)})
 assert get_content_set(od) == ({1, 3}, {2, 4}, {(1, 2), (3, 4)})
+
+d = {}
+assert d.copy() == d
+d = {'a': 1, 'b': 2}
+assert d.copy() == d
```

Comment:
Ideas for additional checks:

* Test that the result is not the same object (`assert d.copy() is not d`).
* Test that this works with `DefaultDict` objects when the annotated type is `Dict[...]`, to check for the non-exact code path here.

Reply:
Added some additional checks. Let me know if it looks ok.

Reply:
Just noticed that these tests are in `driver.py`, which doesn't get compiled, so these aren't actually testing anything useful. The recommended way of writing tests is to avoid `driver.py` altogether so that everything gets compiled, in part because it's easy to accidentally have an essentially no-op test case. Unfortunately, most of the existing test cases use `driver.py`.

Reply:
The test code still needs to be moved away from `driver.py`. It would probably be easier to create a new test case that doesn't define `driver.py` at all (see the mypyc developer docs for how to do this).